github/compliance/model.py

```diff
@@ -46,9 +46,15 @@ class ScanState(enum.Enum):
     FAILED = 'failed'
 
 
+Target = cnudie.iter.ResourceNode | cnudie.iter.SourceNode
+'''
+type of an element to reference the target of a scan result
```

Comment:
this docstr is too short, and gives too little context to actually be useful. I suggest to simply remove it for now, and maybe add a comprehensive documentation, e.g. on module-level, later.